pyanaconda/modules/payload/live/live_image_interface.py

```diff
@@ -76,3 +103,70 @@ def VerifySSL(self) -> Bool:
     def SetVerifySSL(self, verify_ssl: Bool):
         """Set if the ssl verification should be enabled."""
         self.implementation.set_verifyssl(verify_ssl)
+
+    @property
+    def RequiredSpace(self) -> UInt64:
+        """Space required by the source image."""
+        return self.implementation.required_space
+
+    @emits_properties_changed
+    def SetRequiredSpace(self, required_space: UInt64):
+        """Set the space required by the source image."""
+        self.implementation.set_required_space(required_space)
+
+    def UpdateKernelVersionList(self):
+        """Update the list of kernel versions."""
+        self.implementation.update_kernel_version_list()
+
+    def GetKernelVersionList(self) -> List[Str]:
+        """Get the kernel versions list."""
+        return self.implementation.kernel_version_list
+
+    @dbus_signal
+    def KernelVersionListChanged(self, kernel_version_list: List[Str]):
```

Comment:
I thought that given there is no property, I should provide a signal. But yes this can be removed, or replaced by lazy property, or ...
Something to solve in a separate PR?

Reply:
Ok.